learn/guides/running-ballerina-programs-in-the-cloud/function-as-a-service-with-ballerina/aws-lambda.md

```diff
@@ -114,33 +123,73 @@ Execute the command below to test the deployed AWS Lambda function by invoking i
+$ aws lambda invoke --function-name echo --payload fileb://input.json echo-response.txt
 {
     "StatusCode": 200,
     "ExecutedVersion": "$LATEST"
 }
 ```
 
-Execute the command below to view the response in the `response.txt` file.
+Execute the command below to view the response in the `echo-response.txt` file.
 
 ```bash
-cat response.txt 
+$ cat echo-response.txt
+
+{"MESSAGE":"HELLO"}
 ```
 
-You view the output below.
+If you want to check the logs of the execution, you have to go to `echo` lambda function in the portal then click on Monitor tab and Logs button.
 
-```bash
-"dd9446a11b2021b753a5df48d11f339055375b59cd81d7559d36b652aaff849d"
+## More Samples
+
+### S3 Trigger
+In this sample we will be creating a function that will be executed for each object creation in AWS S3.
+
+```ballerina
+import ballerinax/awslambda;
+
+@awslambda:Function
+public function notifyS3(awslambda:Context ctx, 
+                         awslambda:S3Event event) returns json {
+                            io:println (event.Records[0].s3.'object.key);
+    return event.Records[0].s3.'object.key;
+}
 ```
 
-## What's Next?
+Now you can build and deploy the function as we did in previous sample.
+In order for us to invoke this function, we need to create a s3 bucket in AWS.
+1. Go to [AWS S3 Page](https://s3.console.aws.amazon.com/s3/) in the portal and create a bucket
+2. Click on the created bucket and go to Properties tab and click on Create event notification under Event notifications section.
+3. Enable `All object create events under` event types. Select Lambda function as the destination and choose the notifyS3 lambda function from the dropdown.
+
+Now we can click on `Upload` to upload a object to the s3 bucket and view the lambda logs via cloudwatch to see the object name.
+
+### DynamoDB Trigger
+In this sample we will be created a function that will be executed for each entry added to a database in the DynamoDB.
+
+```ballerina
+import ballerinax/awslambda;
+
+@awslambda:Function
+public function notifyDynamoDB(awslambda:Context ctx, 
+                               awslambda:DynamoDBEvent event) returns json {
+                                  io:println (event.Records[0].dynamodb.Keys.toString());
+    return event.Records[0].dynamodb.Keys.toString();
+}
+```
+Now you can build and deploy the function as we did in previous sample.
+In order for us to invoke this function, we need to create a dynamodb table first.
+1. Go to [roles](https://console.aws.amazon.com/iamv2/home#/roles ) and add `AWSLambdaDynamoDBExecutionRole` to the created role in the prerequisites.
+2. Go to [DynamoDB](https://us-west-1.console.aws.amazon.com/dynamodbv2) 
+3. Click on Create Table and enter the table name, partition key and create the table. If you already have a table you can skip this step.
+4. Click on the dynamodb table and go exports and streams tab.
+5. Click on enable DynamoDB stream details and select Key attributes only for the event type.
```

Comment:
```suggestion
5. Click **enable DynamoDB stream details**, and select the key attributes only for the event type.
```
Make sure to have the font of all UI elements in bold. Please check all ossurrences.